packages/salesforcedx-vscode-core/src/notifications/notificationService.ts

```diff
@@ -74,15 +76,31 @@ export class NotificationService {
     observable: Observable<number | undefined>,
     cancellationToken?: vscode.CancellationToken
   ) {
+    const sfdxCoreSettings = SfdxCoreSettings.getInstance();
```

Comment:
We usually do it a different way where we export the instance directly in a index.ts file. See https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/notifications/index.ts#L10